crates/bevy_window/src/window.rs

```diff
@@ -290,6 +289,18 @@ pub enum WindowCommand {
     Close,
 }
 
+/// Defines if and how the cursor is grabbed.
```

Comment:
This should briefly describe how this struct might be used.

This is easy to encounter on docs.rs, but doesn't hint at usage.

Reply:
Great. Added.